### ISSUE TITLE
Cannot pull directly from IDE

### ISSUE THREAD

**Issue by kanokc** [zemismart/zigbee#2]
Cannot pull this handler directly from Samsung IDE.

**Comment by Jhonattan-Souza** [comment]
Hey man, Wesley made an excellent PR that fix this problem, my suggestion is to make a fork in your github. Because @zemismart is not looking at this repository.

**Comment by kanokc** [comment]
> 
> 
> Hey man, Wesley made an excellent PR that fix this problem, my suggestion is to make a fork in your github. Because @zemismart is not looking at this repository.

Yes, that what I currently did.  But I thought that pulling from the original is better.  If Zemismart does not monitor this, will keep getting it from Wesley.

Thanks.

**Comment by zemismart** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        hi dear, what is Wesley?
        
    
        
            
        
    
    
     
    
        
            
    
        
            
                
                    
                        
                    
                    
                        
                            sky
                            
                                    
                                    zemismart smart home supplier
                                  
                                     - 
                                    Promotion Manager
                            
                                
                                    +86153246101220
                                
                                
                                    sky@zhi-mei.com
                                
                                
                                    KeChuang B502，Huifeng 2nd Road No.16 
                                
                                
                                    516006
                                
                                
                                    
                                
                        
                    
                
            
        
    

        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 6/24/2020 17:35，kanokc<notifications@github.com> wrote： 




Hey man, Wesley made an excellent PR that fix this problem, my suggestion is to make a fork in your github. Because @zemismart is not looking at this repository.

Yes, that what I currently did.  But I thought that pulling from the original is better.  If Zemismart does not monitor this, will keep getting it from Wesley.
Thanks.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by w35l3y** [comment]
> what is Wesley?

It is me.

I have sent PR #1 that should fix this issue along with other ones.

**Comment by patricktd** [comment]
This only for app Smartthings Classic. For the new application is
unnecessary.

Em seg, 29 de jun de 2020 05:22, zemismart <notifications@github.com>
escreveu:

**Comment by w35l3y** [comment]
> This only for app Smartthings Classic. For the new application is unnecessary.

What did you mean?
It is a requirement of Samsung Groovy IDE.
It just recognizes repositories with a specific folder structure.
It currently is the common place to add custom DTHs, independently of app version.

**Comment by patricktd** [comment]
In the new application, additional fields are still being released. If you
check, the new DTH do not have custom fields.


Em seg., 29 de jun. de 2020 às 17:12, w35l3y <notifications@github.com>
escreveu:
